contrib/completion/bash/docker

```diff
@@ -2803,7 +2803,7 @@ _docker_image_pull() {
 
 	case "$cur" in
 		-*)
-			local options="--all-tags -a --disable-content-trust=false --help"
+			local options="--all-tags -a --quiet -q --disable-content-trust=false --help"
```

Comment:
nitpicking (not super important): I could be wrong, but as far as I can tell, most of the flags in this script are ordered alphabetically, so I think it would be good to keep the list in alphabetical order.

Reply:
Yes, I believe they are kept in alphabetical order 👍

Reply:
Please update the argument order.